### DISCUSSION TITLE
关于日历组件的更新

### DISCUSSION THREAD

**Discussion by vVvlife** [ant-design/ant-design#32674] [Ideas]
希望能像fullcalendar或者隔壁某音的 semi design那样，可以加入日视图与周视图
对于跨日事项的支持

**Comment by vVvlife** [comment]
日视图：
![image](https://user-images.githubusercontent.com/20148964/139172765-98f2220c-d596-474d-864d-34511d839030.png)

周视图：
![image](https://user-images.githubusercontent.com/20148964/139172721-e7a747ae-1d8f-46b0-aad8-71a7a188d162.png)

跨日事件支持：
![image](https://user-images.githubusercontent.com/20148964/139172817-137f626f-85c9-4d10-89be-973453d42a51.png)

**Comment by DreamPWJ** [comment]
隔壁优秀
